<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BufferMenuTabWidgetClass</class>
 <widget class="QTabWidget" name="BufferMenuTabWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BufferMenuTabWidget</string>
  </property>
  <property name="currentIndex">
   <number>3</number>
  </property>
  <widget class="QWidget" name="viewFrameBufferTab">
   <attribute name="title">
    <string>VIEW FRAME BUFFER</string>
   </attribute>
  </widget>
  <widget class="QWidget" name="readPageTab">
   <attribute name="title">
    <string>READ PAGE</string>
   </attribute>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="readPageFormLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="pageNumberLabel">
       <property name="text">
        <string>Page Number</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="readPageNumberLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="readPagePushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>READ PAGE</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="writePageTab">
   <attribute name="title">
    <string>WRITE PAGE</string>
   </attribute>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>50</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="pageNumberLabel_2">
       <property name="text">
        <string>Page Number</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="writePageNumberLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>SOME DATA TO BE WRITTEN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="writePagePushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>WRITE</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="someDataToBeWritten">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="hitRateTab">
   <attribute name="title">
    <string>HIT RATE</string>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
